templates/vue-starter-template/app/components/FrontendAccountCustomerGroupRegistrationPage.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+const props = defineProps<{
+  navigationId: string;
+}>();
+
+const { apiClient } = useShopwareContext();
+
+const { data: registrationResponse } = await useAsyncData(
+  `cmsNavigation${props.navigationId}`,
+  async () => {
+    const response = await apiClient.invoke(
+      "getCustomerGroupRegistrationInfo get /customer-group-registration/config/{customerGroupId}",
+      {
+        pathParams: { customerGroupId: props.navigationId },
+      },
+    );
+    return response.data || {};
+  },
+);
+</script>
+
+<template>
+  <div class="container mx-auto bg-white flex flex-col">
+    <h1
+      class="mb-4 text-2xl font-extrabold leading-none tracking-tight text-secondary-900 md:text-3xl lg:text-5xl dark:text-white text-center"
+    >
+      {{ registrationResponse?.translated.registrationTitle }}
```

Comment:
we should not have optional chaining here - either we do have the response or not, if not - error page

Reply:
If you check the code below, it is not like that. no response =/= error page. I created a task for that. Component is moved from demo template

https://github.com/shopware/frontends/issues/1845